### PR TITLE
docs: Remove pruned node warning.

### PR DESCRIPTION
**Pull request by sime**
In #5553 `docs/INSTALL.md` was updated to reflect that prune nodes are supported.

#### Pull Request Checklist

- [ ] All changes are Go version 1.15 compliant
- [ ] Your PR passes all CI checks. If a check cannot be passed for a justifiable reason, that reason must be stated in the commit message and PR description.
- [x] If this is your first time contributing, we recommend you read the [Code Contribution Guidelines](https://github.com/lightningnetwork/lnd/blob/master/docs/code_contribution_guidelines.md)
   - [ ] The code being submitted is commented according to [Code Documentation and Commenting](https://github.com/lightningnetwork/lnd/blob/master/docs/code_contribution_guidelines.md#CodeDocumentation)
   - [ ] Commits have a logical structure according to [Ideal Git Commit Structure](https://github.com/lightningnetwork/lnd/blob/master/docs/code_contribution_guidelines.md#IdealGitCommitStructure)
- [ ] For new code: Code is accompanied by tests which exercise both the positive and negative (error paths) conditions (if applicable)
- [ ] For bug fixes: If possible, code is accompanied by new tests which trigger the bug being fixed to prevent regressions
- [ ] Any new logging statements use an appropriate subsystem and logging level
- [x] For code and documentation: lines are wrapped at 80 characters (the tab character should be counted as 8 characters, not 4, as some IDEs do per default)
- [ ] A description of your changes [should be added to running the release notes](https://github.com/lightningnetwork/lnd/tree/master/docs/release-notes) for the milestone your change will land in.

### CODE REVIEW

File: docs/safety.md [hunk 1]
```diff
@@ -20,7 +20,8 @@
   - [Prevent data corruption](#prevent-data-corruption)
   - [Don't interrupt `lncli` commands](#dont-interrupt-lncli-commands)
   - [Regular accounting/monitoring](#regular-accountingmonitoring)
-  - [Pruned bitcoind node](#pruned-bitcoind-node)
+  - [The `-txindex` flag](the--txindex-flag)
```
Comment (guggero) [inline docs/safety.md:23]: The links need a hash/pound (`#`) sign, otherwise they won't work.

**Review by guggero — COMMENTED** [review]
Thanks for the update. See inline comments.

File: docs/safety.md [hunk 2]
```diff
@@ -415,21 +416,14 @@ Regular monitoring of a node and keeping track of the movement of funds can help
...
-not supported! `lnd` needs to verify the funding transaction of every channel
-in the network and be able to retrieve that information from `bitcoind` which
-it cannot deliver when that information is pruned away.
-
-In theory pruning away all blocks _before_ the SegWit activation would work
-as LN channels rely on SegWit. But this has neither been tested nor would it
-be recommended/supported.
+### The `-txindex` flag
 
 In addition to not running a pruned node, it is recommended to run `bitcoind`
```
Comment (guggero) [inline docs/safety.md:421]: Perhaps we should re-word this now? That for performance reasons it is recommended to _not_ run a pruned node if possible and to enable the `-txindex` flag as well?

**Comment by sime** [comment]
@guggero Whoops. My bad, it's sorted now.

**Review by yyforyongyu — CHANGES_REQUESTED** [review]
Thanks for the PR. I think you need to squash your commits into one, and use the `skip ci` tag suggested in our [developer guide](https://github.com/lightningnetwork/lnd/blob/master/docs/code_contribution_guidelines.md#ideal-git-commit-structure).

**Comment by sime** [comment]
@yyforyongyu Squash and appended `[skip ci]`

**Review by guggero — APPROVED** [review]

**Review by yyforyongyu — APPROVED** [review]

**Pull request merged**